controllers/topology/object_builders_test.go

```diff
@@ -500,27 +500,27 @@ func (f *fakeControlPlane) Obj() *unstructured.Unstructured {
 }
 
 type fakeMachineDeployment struct {
-	namespace              string
-	name                   string
-	bootstrapTemplate      *unstructured.Unstructured
-	infrastructureTemplate *unstructured.Unstructured
-	labels                 map[string]string
+	namespace                     string
+	name                          string
+	bootstrapTemplate             *unstructured.Unstructured
+	infrastructureMachineTemplate *unstructured.Unstructured
```

Comment:
Why rename this? Given that's in a MachineDeployment seems pretty straightforward?

Reply:
I kept it that way when I merged my variant of it with the one previously merged on master.

Mainly because it seemed like a good idea to me to keep it consistent with what we already have with `fakeControlPlaneTemplate`. (field as well as the With func)

Reply:
Seems redundant when reading it, might make sense to keep it simple?

Reply:
Makes sense, done